daml-lf/spec/daml-lf-1.rst

```diff
@@ -483,18 +487,38 @@ We now define all the literals that a program can handle::
   Contract ID literals:
         cid   ::= cidV0 | cidV1                      -- LitCid
 
+  Rounding Mode Literals:                            -- LitRoundingMode
+        LitRoundingMode ::=
+           | ROUNDING_UP
+           | ROUNDING_DOWN
+           | ROUNDING_CEILING
+           | ROUNDING_FLOOR
+           | ROUNDING_HALF_UP
+           | ROUNDING_HALF_DOWN
+           | ROUNDING_HALF_EVEN
+           | ROUNDING_UNNECESSARY
+
 The literals represent actual Daml-LF values:
 
 * A ``LitNatType`` represents a natural number between ``0`` and
   ``38``, bounds inclusive.
 * A ``LitInt64`` represents a standard signed 64-bit integer (integer
   between ``−2⁶³`` to ``2⁶³−1``).
-* A ``LitNumeric`` represents a signed number that can be represented
-  in base-10 without loss of precision with at most 38 digits
-  (ignoring possible leading 0 and with a scale (the number of
-  significant digits on the right of the decimal point) between ``0``
-  and ``37`` (bounds inclusive). In the following, we will use
-  ``scale(LitNumeric)`` to denote the scale of the decimal number.
+* A decimal number is a signed number that can be represented as a
+  product `i * 10^-s` where `i` (the *unscaled value* of the number)
+  is a signed integer not divisible by ten and `s` (the *scale* of the
+  number) is a signed integer. The *precision* of a decimal numbers
+  is the number of digits of its unscaled value (ignoring possible
+  leading zeros). By convention the scale and the precision of zero
+  are 0.  Daml-LF distinguishes two kinds of decimal numbers:
+  + A ``LitNumeric`` represents those decimal numbers that have a
+    precision of at most 38 and a scale between ``0`` and ``37``
+    (bounds inclusive).
+  + A ``LitBigNumeric`` represents those decimal numbers smaller than
+    `10^(-MaxScale+1)` that have a scale of at least ``MaxScale``.
```

Comment:
I'm confused because "MaxScale" sounds like an upper bound, but this sentence says it's a lower bound.

```suggestion
    `10^(MaxScale+1)` that have a scale of at most ``MaxScale``.
```

Also shouldn't there be some restriction on the precision?